contracts/TileDiamond/facets/ERC1155TileFacet.sol

```diff
@@ -78,15 +80,15 @@ contract ERC1155TileFacet is Modifiers {
       LibERC1155Tile.addToOwner(_to, id, value);
       IERC1155Marketplace(s.aavegotchiDiamond).updateERC1155Listing(address(this), id, _from);
     }
-    emit LibERC1155Tile.TransferBatch(sender, _from, _to, _ids, _values);
+    emit LibERC1155.TransferBatch(sender, _from, _to, _ids, _values);
     LibERC1155Tile.onERC1155BatchReceived(sender, _from, _to, _ids, _values, _data);
   }
 
   function setApprovalForAll(address _operator, bool _approved) external {
     address sender = LibMeta.msgSender();
     require(sender != _operator, "ERC1155Facet: setting approval status for self");
     s.operators[sender][_operator] = _approved;
-    emit LibERC1155Tile.ApprovalForAll(sender, _operator, _approved);
+    emit LibERC721.ApprovalForAll(sender, _operator, _approved);
```

Comment:
again, why are you calling LibERC721 events in an ERC1155 function?

Reply:
@cinnabarhorse Let me make a new lib for sharing common functions and events.